docs/en/query-builder.md

```diff
@@ -0,0 +1,43 @@
+# Query builder
+
+Built on top of [Yii DB](https://github.com/yiisoft/db), query builder allows you to construct a **SQL query** in a programmatic and **DBMS-agnostic** way. Compared to writing **raw SQL statements**, using query builder will help you write more readable **SQL-related** code and generate more secure **SQL statements**.
+
+Using query builder usually involves two steps:
+
+1. Build a `Yiisoft\Db\Query\Query` class to represent different parts (e.g. `SELECT`, `FROM`) of a `SELECT` **SQL statement**.
+2. Execute a **query method** (e.g. `all()`, `one()`, `scalar()`, `column()`, `query()`) of `Yiisoft\Db\Query\Query` to retrieve data from the database.
+
+The following code shows a typical way of using query builder.
+
+```php
+<?php
+
+declare(strict_types=1);
+
+use Yiisoft\Db\Connection\ConnectionInterface;
+use Yiisoft\Db\Query\Query;
+
+/** @var ConnectionInterface $db */
+
+$rows = (new Query($db))
+    ->select(['id', 'email'])
+    ->from('user')
```

Comment:
from('{{%user}}')

and need add example with correct work with alias
['alias' => '{{%table_name}}']